src/nlp/apply_KB_semisupervised.py

```diff
@@ -0,0 +1,72 @@
+from keybert import KeyBERT
+import nltk
+from nltk.corpus import stopwords
+
+# nltk.download("stopwords")
+# nltk.download("punkt")
+
+# nltk.download("wordnet")
+# nltk.download("omw-1.4")
+from nltk import word_tokenize
+from nltk.stem import WordNetLemmatizer
+import string
+from transformers import AutoTokenizer, AutoModel
+import pandas as pd
+import transformers
+from transformers import pipeline
+
+# apply model semi supervised to text
+def KeywordExtraction(df):
+    tokenizer = AutoTokenizer.from_pretrained(
+        "models/nlp/semi_supervised/model", model_max_lenght=512
```

Comment:
Den file path in einen file "constants.py" hinzufügen und mit os.path.join() erstellen damit er auch auf anderen Computern funktioniert